### ISSUE TITLE
Spades unequal amount of reads

### ISSUE THREAD

**Issue by oyac** [jtamames/SqueezeMeta#770]
Hello, 

I am running my samples in sequential mode, and I did the trimming with trimmomatic first. When it gets to the assembly, with Spades, I get the error " Pair of read files contain unequal amount of reads". Is there a way to fix this, or a workaround?

Thanks!

**Comment by jtamames** [comment]
Hello
Did you run trimmomatic in SqueezeMeta? If so, try running it externally and then provide the resulting fastq. If not, check that you are specifying the correct fastq files.
Best,
J

**Comment by oyac** [comment]
Hi , 
Yes, I used the --cleaning flag so it would run in SqueezeMeta. So, the best option is to run it externally? 
Thanks,

**Comment by fpusan** [comment]
Are the original fastq files properly paired?

**Comment by oyac** [comment]
Yes, the original ones are properly paired.

**Comment by fpusan** [comment]
Then try as Javier suggested and run trimommatic externally. I will leave this open so we remember to look at this potential bug. Can you share your syslog file with us so we can replicate as best as possible?

**Comment by oyac** [comment]
Hi, today i started the project in another folder, from the sample that had gone wrong and it went further without any problems.

Thanks,

**Comment by jtamames** [comment]
Glad to hear! Did you run trimmomatic externally, or in SqueezeMeta as before?
Best,
J

**Comment by oyac** [comment]
I started from zero, running trimmomatic within SqueezeMeta.

Cheers,

**Comment by fpusan** [comment]
Indeed, I just tested `--cleaning` on a sequential run with a mock dataset and got no errors
